### PR TITLE
Add GTN tutorials

### PR DESCRIPTION
**Pull request by supernord**

### CODE REVIEW

File: docs/publications/BioHackrXiv/paper.bib [hunk 1]
```diff
@@ -176,3 +176,37 @@ @article{blankenberg2014dissemination
...
 }
 
+@misc{dev-community-tool-table,
+author = "Bérénice Batut",
+	title = "Creation of an interactive Galaxy tools table for your community (Galaxy Training Materials)",
+	year = "",
+	month = "",
+	day = ""
+	url = "\url{https://training.galaxyproject.org/training-material/topics/dev/tutorials/community-tool-table/tutorial.html}",
+	note = "[Online; accessed Thu Mar 14 2024]"
```
Comment (bebatut) [inline docs/publications/BioHackrXiv/paper.bib:186]:
```suggestion
        author = {Bérénice Batut},
	title = {Creation of an interactive Galaxy tools table for your community (Galaxy Training Materials)},
	year = {2024},
	url = {https://training.galaxyproject.org/training-material/topics/dev/tutorials/community-tool-table/tutorial.html},
	note = {Online; accessed Thu Mar 14 2024}
```

File: docs/publications/BioHackrXiv/paper.bib [hunk 2]
```diff
@@ -176,3 +176,37 @@ @article{blankenberg2014dissemination
...
+}
+
+@misc{dev-tool-annotation,
+author = "Bérénice Batut and Johan Gustafsson and Paul Zierep",
+	title = "Adding and updating best practice metadata for Galaxy tools using the bio.tools registry (Galaxy Training Materials)",
+	year = "",
+	month = "",
+	day = ""
+	url = "\url{https://training.galaxyproject.org/training-material/topics/dev/tutorials/tool-annotation/tutorial.html}",
+	note = "[Online; accessed Thu Mar 14 2024]"
```
Comment (bebatut) [inline docs/publications/BioHackrXiv/paper.bib:211]:
```suggestion
       author = {Bérénice Batut and Johan Gustafsson and Paul Zierep},
	title = {Adding and updating best practice metadata for Galaxy tools using the bio.tools registry (Galaxy Training Materials)},
	year = {2024},
	url = {https://training.galaxyproject.org/training-material/topics/dev/tutorials/tool-annotation/tutorial.html},
	note = {Online; accessed Thu Mar 14 2024}
```

**Review by paulzierep — APPROVED** [review]
Thanks you, looks great !

**Comment by paulzierep** [comment]
if @bebatut agrees I will merge

**Comment by bebatut** [comment]
Sure. Go for it

**Pull request merged**